packages/client/src/object/object.test.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import type { Osdk } from "@osdk/api";
+import type { Osdk, PropertyKeys } from "@osdk/api";
```

Comment:
Why the new import here?

Reply:
accident. can remove in follow up. has no effect on output

Reply:
Sounds good